### PR TITLE
Fixes #2658: Companion release notes before/after new version downloa…

### LINKED ISSUE #2658: Companion release notes before/after new version download broken
When the new release is available and user gets the option to download it, he is asked if he wants to read the release notes. Answering Yes brings up the contributors window and not release notes!

### PR DESCRIPTION
**Pull request by projectkk2glider**
…d broken

Closes #2658

### CODE REVIEW

**Comment by bsongis** [comment]
Thanks, would you press again the button?

**Comment by projectkk2glider** [comment]
Looks like this did not fix it.  I have to check again.

**Comment by bsongis** [comment]
It doesn't compile on Mac, but it should compile nowhere!

File: companion/src/mainwindow.cpp [hunk 1]
```diff
@@ -461,7 +461,7 @@ void MainWindow::checkForFirmwareUpdateFinished(QNetworkReply * reply)
           msgBox.resize(0,0);
           msgBox.exec();
           if( msgBox.clickedButton() == rnButton ) {
-            ContributorsDialog *cd = new ContributorsDialog(this); // , 2, rn);
+            ContributorsDialog *cd = new ReleaseNotesFirmwareDialog(this, rn);
```
Comment (bsongis) [inline companion/src/mainwindow.cpp]: ReleaseNotesFirmwareDialog

**Comment by projectkk2glider** [comment]
Already fixed on master in 572cfd67af3cd2cff59210b2c178fa8719a07ea8

**Comment by projectkk2glider** [comment]
We are good for another release, I will let you do it!

**Comment by projectkk2glider** [comment]
I will make another set of debian packages in the meantime.

**Pull request merged**